packages/fxa-payments-server/src/components/AppLayout/index.scss

```diff
@@ -7,7 +7,7 @@
   background-image: url('./images/firefox-logo.svg');
 }
 
-#main-content.payments-card::before {
+#main-content.payments-card.hide-logo::before {
   background-image: none;
 }
```

Comment:
Yeah, I think your way is the way we're going to need to do this. I do have a recommendation for our CSS though.

Change:
```
#main-content::before {
  background-image: url('./images/firefox-logo.svg');
}

#main-content.payments-card.hide-logo::before {
  background-image: none;
}
```

to:
```
#main-content:not(.hide-logo)::before {
  background-image: url('./images/firefox-logo.svg');
}
```